### PR TITLE
Updated Danish localization

### PR DESCRIPTION
**Pull request by LTank**

### CODE REVIEW

**Comment by corinagum** [comment]
Please add an entry to CHANGELOG.md. Thanks for your contribution!

File: packages/component/src/Localization/da-DK.js [hunk 1]
```diff
@@ -1,23 +1,83 @@
...
   'Tax': 'Skat',
   'Total': 'Total',
   'VAT': 'Moms',
   'Send': 'Send',
-  // 'Speak': '',
-  // 'Upload file': '',
+  'Speak': 'Tal',
+  'Upload file': 'Upload fil',
   'Type your message': 'Skriv din besked'
   // 'X minutes ago':
```
Comment (iMicknl) [inline packages/component/src/Localization/da-DK.js]: Should be changed to: ``` 'X minutes ago': xMinutesAgo```

Comment (corinagum) [reply packages/component/src/Localization/da-DK.js]: please delete this line! Thanks for the catch

File: packages/component/src/Localization/da-DK.js [hunk 2]
```diff
@@ -1,23 +1,83 @@
...
+    });
+  }
+}
+
+
 export default {
-  // FAILED_CONNECTION_NOTIFICATION: '',
+  FAILED_CONNECTION_NOTIFICATION: 'Kunne ikke tilslutte',
   // Do not localize {Retry}; it is a placeholder for "Retry". English translation should be, "Send failed. Retry."
   SEND_FAILED_KEY: 'ikke sendt, {Retry}.',
```
Comment (corinagum) [inline packages/component/src/Localization/da-DK.js]: Should this be `Ikke`? (Capital I)

File: packages/component/src/Localization/da-DK.js [hunk 3]
```diff
@@ -1,23 +1,83 @@
...
+  'Microphone on': 'Mikrofon tændt',
   'Listening…': 'Lytter…',
+  'Left': 'Venstre',
+  'New messages': 'Ny besked',
+  'Right': 'Højre',
   'retry': 'prøv igen',
-  'Retry': '{retry}', // Please alter this value if 'Retry' at the beginning of a sentence is written differently than at the end of a sentence.
-  'Sending': 'sender',
-  // 'Starting…': '',
+  'Retry': '{prøv igen}', // Please alter this value if 'Retry' at the beginning of a sentence is written differently than at the end of a sentence.
```
Comment (corinagum) [inline packages/component/src/Localization/da-DK.js]: Is this supposed to be `Prøv igen`? If so, please update and remove the `{` and `}`

File: packages/component/src/Localization/da-DK.js (hunk 2 above)
Comment (LTank) [reply packages/component/src/Localization/da-DK.js]: No it should definitely be 'Kunne ikke tilslutte'. Thanks for the reviews!

File: packages/component/src/Localization/da-DK.js (hunk 3 above)
Comment (LTank) [reply packages/component/src/Localization/da-DK.js]: I deleted the { and }.

**Pull request merged**